src/docker.rs

```diff
@@ -0,0 +1,139 @@
+use std::collections::HashSet;
```

Comment:
Silly question, but since this file seems to be used only in `src/build_platform/local_docker.rs`, would it makes sense to put it inside the same directory `src/build_platform/`?

Reply:
I was wondering the same. It makes sense for me to put it in the `build_platform` module and move it down if it could be used elsewhere in the future 👍